### DISCUSSION TITLE
EFOB problem (maybe because of EXPERIMENTAL?)

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6012] [Q&A] [answered]
Hello 

I was doing a flight from EWR to LAS with the A32NX (EXPERIMENTAL VERSION) I planned my flight with simbrief. 
When I enter all the data on the ground, I will get a EFOB of xx tons @ DEST. Its very realistic and nearly same as in the Simbrief flightplan. (I use the corrected simbrief profile)
But after takeoff, the EFOB value continuously decrease, and the UTC (ARR TIME) continuously rise in the FUEL PRED page. But on the FPLN page, the UTC ARR time is unchanged. 
So when I arrived @ LAS, my EFOB shows a value of -8.9t. But the FOB was very close to the planned  value.

Is it a problem because of the Experimental version with the flight plan manager? Or is the problem on my side?

thanks for your help

**Comment by tracernz** [comment, marked as answer]
Yeah, FUEL PRED doesn't work properly with the new flight planning system. Will be fixed in future when VNAV can predict fuel.
